Plant phenomics assembly

erlefloch opened the pull request:
Hi,
This is a first version of a plant phenomics assembly page, made thanks to the contributors presenting the different tools. 
I would like to talk about it at IPPS next week.
Feel free to edit or comment,
Cheers
Erwan

bedroesb commented on pages/tool_assembly/plant_phenomics_assembly.md:
```diff
@@ -0,0 +1,131 @@
+### Data processing and analysis 
+
+It is important to keep in mind the difference between data processing and analysing.
+[Processing](https://rdmkit.elixir-europe.org/processing) provides the tools and procedures to transform primary data, such as imaging or observational data, to appropriate quality and processability:  
```
```suggestion
[Processing](processing) provides the tools and procedures to transform primary data, such as imaging or observational data, to appropriate quality and processability:  
```

bedroesb commented on pages/tool_assembly/plant_phenomics_assembly.md:
```diff
@@ -0,0 +1,131 @@
+
+It is important to keep in mind the difference between data processing and analysing.
+[Processing](https://rdmkit.elixir-europe.org/processing) provides the tools and procedures to transform primary data, such as imaging or observational data, to appropriate quality and processability:  
+[Analysis](https://rdmkit.elixir-europe.org/analysing), on the other hand, is concerned with extracting information from the processed data for the purpose of supporting knowledge acquisition: 
```
```suggestion
[Analysis](analysing), on the other hand, is concerned with extracting information from the processed data for the purpose of supporting knowledge acquisition: 
```

bedroesb commented on pages/tool_assembly/plant_phenomics_assembly.md:
```diff
@@ -0,0 +1,131 @@
+
+### Data preservation
+
+No central, global repository with long term [data preservation](https://rdmkit.elixir-europe.org/preserving) exists for Plant Phenotyping data. This is under the responsibility of each country.
```
```suggestion
No central, global repository with long term [data preservation](preserving) exists for Plant Phenotyping data. This is under the responsibility of each country.
```

bedroesb reviewed (commented):
Good first attempt! 

General comment: internal RDMkit pages are linked to by just using the page name as you will see in my suggestions. Can you also look into tagging the tools you mention in the page with the page_id you propose [here](https://docs.google.com/spreadsheets/d/16RESor_qQ_ygI0lQYHR23kbZJUobOWZUbOwhJbLptDE/edit#gid=268211668)

bedroesb commented:
PIPPA btw is part of the Belgian page and should get tagged in the front matter of the national markdown, I don't know for PHIS ;)

erlefloch commented on pages/tool_assembly/plant_phenomics_assembly.md:
```diff
@@ -0,0 +1,131 @@
+related_pages: 
+  your_tasks: [metadata, data publication]
+  your_domain: [plants]
+faircookbook:
```
alright

erlefloch commented on pages/tool_assembly/plant_phenomics_assembly.md:
```diff
@@ -0,0 +1,131 @@
+
+#### File based systems
+
+##### FAIRDOM-Seek
```
I will try with bold links

erlefloch commented on pages/tool_assembly/plant_phenomics_assembly.md:
```diff
@@ -0,0 +1,131 @@
+
+## What is the plant phenomics tool assembly and who can use it ?
+
+The plant phenomics tool assembly covers the whole [life cycle](https://rdmkit.elixir-europe.org/data_life_cycle) of experimental plant phenotyping data.  It uses the concepts of the [MIAPPE](https://www.miappe.org/) (Minimum Information About a Plant Phenotyping Experiment) standard. A general introduction of MIAPPE concepts can be found in the following [webinar](https://tess.elixir-europe.org/ ...
```
that's right, do you mean to cross reference to the #more-information section?

erlefloch replied:
I removed it, it was indeed redundant

erlefloch commented on pages/tool_assembly/plant_phenomics_assembly.md:
```diff
@@ -0,0 +1,131 @@
+
+The plant phenomics tool assembly covers the whole [life cycle](https://rdmkit.elixir-europe.org/data_life_cycle) of experimental plant phenotyping data.  It uses the concepts of the [MIAPPE](https://www.miappe.org/) (Minimum Information About a Plant Phenotyping Experiment) standard. A general introduction of MIAPPE concepts can be found in the following [webinar](https://tess.elixir-europe.org/ ...
+
+The plant phenomics tool assembly helps [researchers](https://rdmkit.elixir-europe.org/researcher), [research data stewards](https://rdmkit.elixir-europe.org/data_steward_research) and [developers in charge of infrastructures](https://rdmkit.elixir-europe.org/data_steward_infrastructure) supporting plant phenotyping data management  to enable:
```
would you remove the links, or the sentence ?

erlefloch replied:
I simplified the sentence as all the roles were present indeed.

erlefloch commented:
> PIPPA btw is part of the Belgian page and should get tagged in the front matter of the national markdown, I don't know for PHIS ;)

do you mean, PHIS should be added to the tools and resources section on the French page ?

erlefloch commented:
Thanks for your feedback @bedroesb and @korbinib !
I modified the text accordingly. I also updated the tool_and_resource_list sheet, and I added the info to link to MIAPPE training material in TeSS.

bedroesb commented:
> > PIPPA btw is part of the Belgian page and should get tagged in the front matter of the national markdown, I don't know for PHIS ;)
> 
> do you mean, PHIS should be added to the tools and resources section on the French page ?

If not already done, yes! :)

erlefloch commented:
hi everyone, sorry I messed up the review requests by clicking everywhere, sorry and don't pay attention :)

bedroesb commented:
@erlefloch is PHIS not build to serve the French institutes like PIPPA, or am I wrong? Otherwise I would move it out of the main tools table to the country page metadata instead of linking towards it.

bedroesb commented:
I also took PIPA out of the main tools table since it is already described on our national page

erlefloch commented:
> @erlefloch is PHIS not build to serve the French institutes like PIPPA, or am I wrong? Otherwise I would move it out of the main tools table to the country page metadata instead of linking towards it.

From what I understood, PHIS is the name of both the software and of an instance of it. The software itself can be deployed anywhere, for example I've seen recently [here](https://www.npec.nl/phenotyping-modules/module-7-data/) that it was in use in Netherlands

bedroesb commented:
@erlefloch ok fair enough! For me it looks good now.

bedroesb commented:
@korbinib This page would be presented this Monday and it would be nice if a first version could be merged. I don't know If you think your comments got implemented and this disserves your approval ? :) Later on you might want to apply your tricks to include the font in the SVG

bedroesb commented:
@erlefloch I just clicked on the GitHub icon on the PHIS website, and it goes to [Open SILEX](https://github.com/OpenSILEX/). Maybe PHIS is a deployment of  OpenSIlex?

korbinib commented:
@bedroesb @erlefloch  Thank you!

erlefloch commented:
> @erlefloch I just clicked on the GitHub icon on the PHIS website, and it goes to [Open SILEX](https://github.com/OpenSILEX/). Maybe PHIS is a deployment of OpenSIlex? That one for sure seems to disserve a spot in the main tools table.

Yes probably ! @cpommier what was the relationship between OpenSILEX and PHIS again ?

bedroesb commented:
I am intrested in the answer for sure ;) Will upload the tools to RDMkit as they are now

cpommier commented:
HI, sorry for the delayed answer and thanks a lot for your  high activity :)
Opensilex is the basis software, that can be implemented in several flavors: PHIS, SIXTINE, VitisOno, etc...  
Then each of those flavors are deployed on different sites. 
PHIS is a flavor dedicated to high throughput phenotyping, like PIPPA, but is getting used beyond France. 
If I understand the logic well, we should put forward PHIS in the general tools section as it goes beyond France. Linking this to phis website, that links to opensilex github, seems ok to me. @Isabelle-inrae could you confirm that this is fine with you ?

cpommier commented:
This has ben merged ? It is therefore live now ?

bedroesb commented:
@cpommier when #994 is merged it will be live in the way you described! I just wonder if it wis worthed to mention Opensilex somewhere on RDMkit. I never heard of it.

Isabelle-inrae commented:
Hi @bedroesb, I confirm @cpommier answer; I think it's better to put PHIS forward! Thanks

bedroesb commented:
Thanks for responding!
